Doc/library/trace.rst

```diff
@@ -13,6 +13,10 @@ annotated statement coverage listings, print caller/callee relationships and
 list functions executed during a program run.  It can be used in another program
 or from the command line.
 
+.. seealso:: `Coverage.py <https://coverage.readthedocs.io/>`_, which is one
```

Comment:
Looking at the rest of the documentation is seem like most `.. seealso::` are alone on their lines with the paragraph on the next one.